### DISCUSSION TITLE
how to add icon inside of canvas?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4347] [Q&A]
Hello @artf  thank for this amazing features.
 can you tell me how to add icons inside of canvas and how to make image as a clickable?

**Comment by lexoyo** [comment]
Hello
I will start a FAQ for users
Are you using the online demo?
You need to drop a Link Block on the stage, then the image inside
Icons are images right?

**Reply by senthil177cse** [reply]
no like font awsome icons need to import in canvas...

**Comment by lexoyo** [comment]
You probably need to follow the instructions on font awesome website

**Comment by Ju99ernaut** [comment]
You can also try looking into [iconify](https://iconify.design/)

**Comment by senthil177cse** [comment]
@artf  any comments from your side please?

**Reply by senthil177cse** [reply]
Hi @artf ...
    how to add icons inside of canvas? can you please help me on that?..

**Reply by artf** [reply]
What kind of icons @senthil177cse? Can you be more specific?

**Reply by IndrajeethY** [reply]
> What kind of icons @senthil177cse? Can you be more specific?

i want to include some icons in react-icons

**Reply by mosh-tudor** [reply]
@TAMILVIP007 So what's the problem? You can use SVG from react-icons.

_But it would be better not to engage in necrophilia and to open a new topic when you have a problem._
